asr1k_neutron_l3/plugins/l3/service_plugins/l3_extension_adapter.py

```diff
@@ -353,8 +368,45 @@ def get_sync_data(self, context, router_ids=None, active=None, host=None):
                 all_port_ids = [port["id"] for port in all_ports]
                 router["fwaas_policies"] = self.get_fwaas_policies(context, all_port_ids)
 
+            router["vpn"] = None
+            if utils.is_vpnaas_enabled() and router["flavor_id"]:
```

Comment:
[nit] We could store this in a variable ``vpnaas_enabled = utils.is_vpnaas_enabled()`` in the constructor. In another method of this class the same function was called.